web_src/js/components/ContextPopup.vue

```diff
@@ -91,16 +91,16 @@ export default {
 <template>
   <div ref="root">
     <div v-if="loading" class="tw-h-12 tw-w-12 is-loading"/>
-    <div v-if="!loading && issue !== null">
-      <p><small>{{ issue.repository.full_name }} on {{ createdAt }}</small></p>
-      <p><svg-icon :name="icon" :class="['text', color]"/> <strong>{{ issue.title }}</strong> #{{ issue.number }}</p>
-      <p>{{ body }}</p>
+    <div v-if="!loading && issue !== null" class="tw-flex tw-flex-col tw-gap-2">
+      <div class="tw-text-12">{{ issue.repository.full_name }} on {{ createdAt }}</div>
+      <div><svg-icon :name="icon" :class="['text', color]"/> <strong>{{ issue.title }}</strong> #{{ issue.number }}</div>
+      <div v-if="body">{{ body }}</div>
       <!-- eslint-disable-next-line vue/no-v-html -->
-      <div v-html="renderedLabels"/>
+      <div v-if="(issue?.labels || []).length" v-html="renderedLabels"/>
     </div>
     <div v-if="!loading && issue === null">
-      <p><small>{{ i18nErrorOccurred }}</small></p>
-      <p>{{ i18nErrorMessage }}</p>
+      <div class="tw-text-12">{{ i18nErrorOccurred }}</div>
+      <div>{{ i18nErrorMessage }}</div>
```

Comment:
Why not use `p`?

Reply:
Because `p` style as margin-bottom which would result in unequal margin rendering. `gap` does not have this issue.

Reply:
Fixed in https://github.com/go-gitea/gitea/pull/30824/commits/e09fe684f5da8311ad0234a3780060546e3c49c9.

Reply:
But for this case, you forgot the space between these elements ....

I think `<p>` is good enough for most cases, and no need to use these `tw-` helpers.

Reply:
If there are more cases for `tw-flex tw-flex-col tw-gap-xxx`, I would suggest to add some general classes for it.

Reply:
Yeah, it'd basically be the same as `flex-text-block` but vertical. But overall I'm not convinced it's a good idea to introduce such helpers. As seen by `flex-text-block` having uncertainty regarding overflow etc.

Reply:
Let's leave the `tw-flex tw-flex-col tw-gap-2`. It's not so much to write and we have full flexibility. Sometimes, different gap will be needed etc.